packages/editor-ui/src/components/InlineExpressionEditor/InlineExpressionEditorInput.vue

```diff
@@ -25,14 +25,14 @@ type Props = {
 	modelValue: string;
 	path: string;
 	rows?: number;
-	isReadonly?: boolean;
+	isReadOnly?: boolean;
 	additionalData?: IDataObject;
 	eventBus?: EventBus;
 };
 
 const props = withDefaults(defineProps<Props>(), {
 	rows: 5,
-	isReadonly: false,
+	isReadOnly: false,
```

Comment:
Any idea why this wasn't caught build time by e.g. typescript?

Reply:
I'm not sure, it seems inconsistent: for some components I get TypeScript errors and others I don't, even in very similar components:

<img width="554" alt="image" src="https://github.com/n8n-io/n8n/assets/8850410/5f4a8e8f-a685-4ca8-9b96-a04c0b5645d1">
<img width="554" alt="image" src="https://github.com/n8n-io/n8n/assets/8850410/09a7ca04-c53c-4911-8468-5083772b74bf">

Reply:
we unfortunately do not have build-time type checking yet.